populator-machinery/controller.go

```diff
@@ -468,9 +465,10 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 
 	// Overriding the namespace to our target namespace
 	var secretName string
+	var populatorNamespace string
```

Comment:
no need for the comment above now (and it's incorrect..)

Reply:
removed